float32 temperature
string unit
int64 timestamp
